soda/src/main/java/it/okkam/opendata/models/soda/SodaClassification.java

```diff
@@ -9,6 +9,7 @@
   String domain_category;
   String[] domain_tags;
   Map<String, String>[] domain_metadata;
+  Map<String, String>[] domain_private_metadata;
 
   public String[] getCategories() {
     return categories;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') it.okkam.opendata.models.soda.SodaClassification.getCategories() may expose internal representation by returning SodaClassification.categories [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=findbugs%3AEI_EXPOSE_REP)